### PR TITLE
Prepare release 0.3.0

### PR DESCRIPTION
**Pull request by rubenvp8510**

### CODE REVIEW

**Comment by rubenvp8510** [comment]
Ran openshift e2e tests:

`--- PASS: kuttl (175.88s)
    --- PASS: kuttl/harness (0.00s)
        --- PASS: kuttl/harness/route (23.39s)
        --- PASS: kuttl/harness/multitenancy (140.85s)
        --- PASS: kuttl/harness/monitoring (168.81s)
PASS
➜  tempo-operator git:(release-0.3.`

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,58 @@ Changes by Version
...
+  
+- `operator`: Use internal certificate for internal HTTP server of gateway (#480)
+- `operator`: Add ability to create and configure route or ingress for the gateway (#265)
+- `operator`: The operator is now at Level 4 - Deep Insights (#504)
+  The operator optionally exposes metrics and alerts for the operator and the operand.
+  
+- `operator`: Enable mTLS on receivers when gateway is enabled (#535)
+- `operator`: Enable multitenancy without need the gateway (#224)
+- `operator`: Add operator alerts and runbook. (#309)
+  The alerts are only installed for the OpenShift bundle.
```
Comment (andreasgerstmayr) [inline CHANGELOG.md:34]:
```suggestion

```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,58 @@ Changes by Version
...
+  See https://github.com/grafana/tempo-operator/issues/491 for a discussion to enable them in the community bundle.
```
Comment (andreasgerstmayr) [inline CHANGELOG.md:35]:
```suggestion

```

Comment (andreasgerstmayr) [reply CHANGELOG.md:35]: Let's remove these two lines, as another change in the same release allows enabling them for the community bundle with just an operator configuration setting :)

**Review by andreasgerstmayr — APPROVED** [review]

**Review by pavolloffay — APPROVED** [review]

**Comment by andreasgerstmayr** [comment]
Unit tests fail because `ensure-generate-is-noop` fails because it assumes `0.3.0` is already tagged (can be ignored for this PR).

**Pull request merged**